test/pkcs12_format_test.c

```diff
@@ -733,6 +791,10 @@ int setup_tests(void)
     ADD_TEST(test_cert_key_with_attrs_and_mac);
     ADD_TEST(test_cert_key_encrypted_content);
     ADD_TEST(test_single_secret_encrypted_content);
+    if (lgcyprov == NULL)
```

Comment:
I would probably move this up to group it with the other test case that is like this.